config/locales/de.yml

```diff
@@ -246,6 +246,9 @@ de:
     individual_entry_threatise_printed: "Theoreticum, gedruckt - Teileintrag der Sammlung"
     single_threatise_printed: "Theoreticum, gedruckt - Individualeintrag"
   
+    legacy_templates: "LEGACY TEMPLATES"
```

Comment:
The locales are updated upstream in https://github.com/rism-digital/translations. Any changes here will be overwritten when we next sync them up.

Also, we are trying to not put "placeholders" in the language files, so every new entry should be translated into its appropriate language before being added.

Reply:
I just followed @xhero advice: https://github.com/rism-ch/muscat/pull/1017#issuecomment-721639815

So, which is the preferred way now?